.github/.licenserc.yaml

```diff
@@ -14,5 +14,6 @@ header:
     - 'NOTICE'
     - '.asf.yaml'
     - '.nojekyll'
+    - 'Package.swift'
```

Comment:
This file has the required ASF license header. However, the first line of this file should be `swift-tools-version: 6.0`. So, I added here inevitably.